src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -8,6 +8,7 @@ import cromwell.engine.backend.jes.Run.{Failed, Running, Success, _}
 import cromwell.engine.db.{JesStatus, JesId, JesCallBackendInfo, DataAccess}
 import cromwell.util.google.GoogleScopes
 import org.slf4j.LoggerFactory
+import DataAccess._
```

Comment:
It's somewhat ambiguous I think because you indeed only need to depend on the `DataAccess` trait but when you import `DataAccess._` you actually make a dependency to Slick since the implementation of the trait is a `new SlickDataAccess()` in the `DataAccess` object.
Even if in most cases I think it overcomplexifies everything, IMO this is a case where the "cake pattern" method is pretty cool when you can say 
```scala
class Run { self: DataAccessDependency =>
...
}
```
with 
```scala
trait DataAccessDependency {
  val dataAccess: DataAccess // DataAccess trait
}
```
and then mix anything you want upon instantiation
```scala
trait SlickDataAccess extends DataAccessDependency {
  val dataAccess = new SlickDataAccess()
}
new Run(...) with SlickDataAccess
```
This makes it clear in the `Run` class that you only want "some kind of implementation for DataAccess", and let you choose at instantiation time which one you want to use.

Anyway that's probably out of scope for this ticket, changing to `globalDataAccess` should do it.